frontend/src/app/commonComponents/Alert.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import classnames from "classnames";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
Browser can generate uuid natively `crypto.randomUUID()` https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

Reply:
great suggestion, but not sure if its easy to incorporate, the crypto module will require a pollyfill for nodejs. Kinda leaning towards keeping the uuid since thats already included in the app

![image](https://github.com/CDCgov/prime-simplereport/assets/4952042/fb8e1423-2194-49bf-ae8f-4ae8be9c09ac)

Reply:
`crypto` is part of the windows object; it is already [used](https://github.com/CDCgov/prime-simplereport/blob/927dcb89a870cf148bad95a1f6926f3af56a2850/frontend/src/app/patients/UploadPatients.tsx#L112) in the app. Wonder if you were importing it and that caused the exception 🤔 .

Reply:
yeah it totally works in the browser, just not in a nodejs env such as unit tests. I suspect that referenced code is not triggered during unit tests, but I could be wrong

Reply:
Ill try not importing it and report back

Reply:
Ok I removed it as suggested. The issue was the import. But its not available during test, but I got around the issue by mocking it for this SRToastTest. However, as you can see, all frontend tests that use the Alert will need to define an implementation for the crypto module everywhere since there is no module during test
![image](https://github.com/CDCgov/prime-simplereport/assets/4952042/edbf8ac4-c4a9-4a55-b966-92f03d5f8385)

Reply:
![image](https://github.com/CDCgov/prime-simplereport/assets/4952042/257db517-4234-4615-8fec-0184592d779c)
I think we should just use the uuid module everywhere, I wonder if it already uses the crypto api in the browser